auditing_standards.md

```diff
@@ -237,3 +237,73 @@ Crates with this criteria contain very dangerous unsafe rust code.
     *   Undefined behavior may be possible under common circumstances.
 *   Most crates that try to be sound but don't quite make the cut go in `ub-risk-3`, not here. These crates are wildly unsafe, and the only time we should use them is when they are a necessary evil.
 *   Everything worse than `ub-risk-3` goes in here and we should do our best to avoid using them.
+
+## Criteria-agnostic guidelines
+
+### Delta audits should describe the final version
+
+In general, delta audits must never lower the `UB-risk-N` value from the
```

Comment:
I disagree: delta audits can lower the UB risk if the delta fixed _all_ previous delta issues.

I think we should say that a review performed of _just_ a delta should never lower the risk. The risk may be lowered by looking at the delta _and_ the issues found in the previous review.

Reply:
Good catch - makes sense.  I tried to reword this paragraph to capture your point.